### PR TITLE
updated Network.js

### PR DESCRIPTION
**Pull request by arjunvijayvargiya**
reindented.Fix for issue https://github.com/privly/privly-android/issues/41

### CODE REVIEW

**Comment by arjunvijayvargiya** [comment]
Hi @smcgregor ,
                           Something wrong happened with pull request from my branch.Please see this once

**Comment by smcgregor** [comment]
This is still poorly nested.

**Comment by arjunvijayvargiya** [comment]
I will see to it once again

**Comment by arjunvijayvargiya** [comment]
Hi @smcgregor ,
Please see the latest commit

File: shared/javascripts/network_service.js [hunk 1]
```diff
@@ -433,20 +433,24 @@ var privlyNetworkService = {
...
-   */
-  showLoggedOutNav: function() {
-    
-    // Don't show the nav at all if the content is injected.
-    if(typeof privlyHostPage !== "undefined" && privlyHostPage.isInjected()) {
-      return;
+ /**
+ * Show/hide the appropriate navigation items for when the user is logged out.
+ */
+showLoggedOutNav: function() {
```
Comment (smcgregor) [inline shared/javascripts/network_service.js]: You are still improperly formatting the whole block. See how you changed existing lines with only whitespace changes?

**Pull request closed without merging**